rotkehlchen/tests/unit/decoders/test_convex.py

```diff
@@ -88,8 +90,18 @@ def test_convex_pools(ethereum_inquirer):
 
 
 @pytest.mark.vcr()
+@pytest.mark.parametrize('have_decoders', [True])
+@pytest.mark.parametrize('legacy_messages_via_websockets', [True])
 @pytest.mark.parametrize('ethereum_accounts', [[string_to_evm_address('0xC960338B529e0353F570f62093Fd362B8FB55f0B')]])  # noqa: E501
-def test_booster_deposit(database, ethereum_inquirer, ethereum_accounts):
+def test_booster_deposit(
+        rotkehlchen_api_server: 'APIServer',
```

Comment:
why do you need the server here?

Reply:
Due to this https://github.com/rotki/rotki/blob/2ed321bc04b9d1ed15e8f5797cff62fb6a136001/rotkehlchen/tests/utils/database.py#L76.

websocket connection feature needs `rotkehlchen_api_server` fixture, so we can't use a separate database fixture

Reply:
Then this is not a unit test. You pointed at a test assertion. This tells me this is fixable and you need to find a way to  make this into a light unit test.